README instructions for building with skipDocker failing

Running `mvn clean install -DskipDocker` as per the README instructions fails because some tests still rely on Docker.

Can these instructions be updated in the README or the tests relying on docker skipped when `skipDocker` is passed?

Traceback:
```
[ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.12 s <<< FAILURE! - in com.adobe.testing.s3mock.testcontainers.S3MockContainerJupiterTest
[ERROR] com.adobe.testing.s3mock.testcontainers.S3MockContainerJupiterTest.shouldUploadAndDownloadObject  Time elapsed: 0.056 s  <<< ERROR!
org.testcontainers.containers.ContainerLaunchException: Container startup failed
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=adobe/s3mock:2.3.3-SNAPSHOT, imagePullPolicy=DefaultPullPolicy())
Caused by: com.github.dockerjava.api.exception.NotFoundException:
Status 404: {"message":"manifest for adobe/s3mock:2.3.3-SNAPSHOT not found: manifest unknown: manifest unknown"}


[ERROR] com.adobe.testing.s3mock.testcontainers.S3MockContainerJupiterTest.defaultBucketsGotCreated  Time elapsed: 0.057 s  <<< ERROR!
org.testcontainers.containers.ContainerLaunchException: Container startup failed
Caused by: org.testcontainers.containers.ContainerFetchException: Can't get Docker image: RemoteDockerImage(imageName=adobe/s3mock:2.3.3-SNAPSHOT, imagePullPolicy=DefaultPullPolicy())
Caused by: com.github.dockerjava.api.exception.NotFoundException:
Status 404: {"message":"manifest for adobe/s3mock:2.3.3-SNAPSHOT not found: manifest unknown: manifest unknown"}


[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   S3MockContainerJupiterTest.defaultBucketsGotCreated » ContainerLaunch Containe...
[ERROR]   S3MockContainerJupiterTest.shouldUploadAndDownloadObject » ContainerLaunch Con...
[ERROR]   S3MockContainerManualTest.setUp:34 » ContainerLaunch Container startup failed
[ERROR]   S3MockContainerManualTest.setUp:34 » ContainerLaunch Container startup failed
[INFO]
[ERROR] Tests run: 4, Failures: 0, Errors: 4, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] S3Mock - Parent .................................... SUCCESS [  3.754 s]
[INFO] S3Mock - Build Configuration ....................... SUCCESS [  0.730 s]
[INFO] S3Mock - Server .................................... SUCCESS [ 11.487 s]
[INFO] S3Mock - Testsupport ............................... SUCCESS [  0.118 s]
[INFO] S3Mock - Testsupport - Common ...................... SUCCESS [  5.891 s]
[INFO] S3Mock - Testsupport - JUnit4 ...................... SUCCESS [  5.471 s]
[INFO] S3Mock - Testsupport - JUnit5 ...................... SUCCESS [  7.943 s]
[INFO] S3Mock - Testsupport - TestNG ...................... SUCCESS [  4.903 s]
[INFO] S3Mock - Docker .................................... SUCCESS [  0.471 s]
[INFO] S3Mock - Testsupport - Testcontainers .............. FAILURE [  4.601 s]
[INFO] S3Mock - Integration Tests ......................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 47.962 s
[INFO] Finished at: 2021-12-29T20:02:04Z
[INFO] Final Memory: 117M/468M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:3.0.0-M5:test (default-test) on project s3mock-testcontainers: There are test failures.
[ERROR]
[ERROR] Please refer to /local/home/aljohri/non-brazil/S3Mock/testsupport/testcontainers/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :s3mock-testcontainers
```

--- a/testsupport/pom.xml
+++ b/testsupport/pom.xml
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-     Copyright 2017-2021 Adobe.
+     Copyright 2017-2022 Adobe.
 
      Licensed under the Apache License, Version 2.0 (the "License");
      you may not use this file except in compliance with the License.
@@ -35,7 +35,6 @@
         <module>junit4</module>
         <module>junit5</module>
         <module>testng</module>
-        <module>testcontainers</module>
     </modules>
 
     <build>
@@ -57,4 +56,18 @@
           </plugin>
         </plugins>
     </build>
+  <profiles>
+    <profile>
+      <id>build-testcontainers-docker-image</id>
+      <activation>
+        <property>
+          <!-- If running with -DskipDocker, this profile won't be active. -->
+          <name>!skipDocker</name>
+        </property>
+      </activation>
+      <modules>
+        <module>testcontainers</module>
+      </modules>
+    </profile>
+  </profiles>
 </project>

